dev/k3d-act-clean

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
```suggestion
set -eu
```

Reply:
Should not be added. There are optional environment variables in it as well.